vendor/vendor.json

```diff
@@ -341,6 +341,14 @@
 			"versionExact": "v1.10.51"
 		},
 		{
+			"checksumSHA1": "korrMGve3O88cRKjQ+27MK4CqAM=",
+			"path": "github.com/aws/aws-sdk-go/service/cloudsearch",
+			"revision": "124a6dabcd822d9b88383d8e2186e696a12d791d",
+			"revisionTime": "2017-09-13T19:04:32Z",
+			"version": "v1.10.44",
+			"versionExact": "v1.10.44"
+		},
+		{
```

Comment:
Do you think you could extract all the vendor logic to another PR? would help making the review easier, and be in sync with another resources being created, along with the SDK version being regularly updated! 😄